README.md

```diff
@@ -400,14 +399,57 @@ Gradle allows you to selectively run tests. For example, the following
 command runs tests which match the given pattern and excludes all
 others.
 ```
-./gradlew :tests:test --tests *ActionLoopContainerTests*
+./gradlew :tests:test --tests Python*Tests
 ```
 
 ## Python 3 AI Runtime
-This action runtime enables developers to create AI Services with OpenWhisk. It comes with preinstalled libraries useful for running machine learning and deep learning inferences. [Read more about this runtime here](./core/python3AiActionLoop).
+This action runtime enables developers to create AI Services with OpenWhisk. It comes with preinstalled libraries useful for running machine learning and deep learning inferences. [Read more about this runtime here](./core/python3AiAction).
 
 ## Import Project into IntelliJ
 
 Follow these steps to import the project into your IntelliJ IDE.
 - Import project as gradle project.
 - Make sure the working directory is root of the project/repo.
+
+# Using extra libraries
+
+If you need more libraries for your python action,  you can include a virtualenv in the zip file of the action.
+
+The requirement is that the zip file must have a subfolder named `virtualenv` with a script `virtualenv\bin\activate_this.py` working in an Linux Amd64 environment. It will be executed at start time to use your extra libraries.
```

Comment:
```suggestion
The requirement is that the zip file must have a subfolder named `virtualenv` with a script `virtualenv\bin\activate_this.py` working in an Linux AMD64 environment. It will be executed at start time to use your extra libraries.
```